sandbox/apparmor/apparmor.go

```diff
@@ -396,6 +398,30 @@ func probeParserFeatures() ([]string, error) {
 	return features, nil
 }
 
+func systemAppArmorLoadsSnapPolicy() bool {
+	// on older Ubuntu systems the system installed apparmor may try and
+	// load snapd generated apparmor policy (LP: #2024637)
+	f, err := os.Open(filepath.Join(dirs.GlobalRootDir, "/lib/apparmor/functions"))
+	if err != nil {
+		logger.Debugf("cannot open apparmor functions file: %v", err)
+		return false
```

Comment:
for clarity, this is optimistic, it's probably fine given the nature of the checks but wanted to point this out